src/components/Carousel/CarouselDots.tsx

```diff
@@ -1,11 +1,15 @@
 import classNames from 'classnames';
 import { EmblaCarouselType } from 'embla-carousel';
-import { ComponentProps, forwardRef, useCallback, useEffect, useState, useId, useRef, useMemo } from 'react';
+import { ComponentProps, forwardRef, useCallback, useEffect, useState, useRef, useMemo } from 'react';
 import { useCarousel } from './utils';
 import { getCommonProps } from '../../utils';
 import { CarouselDot } from './CarouselDot';
 
 export interface CarouselDotsProps extends ComponentProps<'div'> {
+  /**
+   * A custom `id` for the `<CarouselDots>`
```

Comment:
can we change the document  "custom `id`" to "unique id" for these components please @zhenpanda